tests/x11/remote_desktop/windows_network_setup.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Remote Login: Windows access SLES over RDP
```

Comment:
That is just the same summary as above. It should differ

Reply:
Updated.